packages/test/test-service-load/src/testConfigFile.ts

```diff
@@ -25,6 +25,12 @@ export interface ILoadTestConfig {
      * Required for GC tests. The time after which unreferenced objects become inactive.
      */
     inactiveTimeoutMs?: number;
+    /**
+     * Optional for GC tests. If this is present, once all containers complete the test, they will be re-opened
+     * after this delay and will re-run the test. This it to test scenarios where a document is at rest for a while
+     * and re-opened - Basically, while the document is at rest, objects become inactive or sweep ready.
+     */
+    rerunDelayMs?: number;
```

Comment:
this feel like an odd way to accomplish this. why do all containers need to close? may will close and re-open over the run? how is that different? and how do we determine this value to get the behavior you describe? Will it always work without additional assumptions?

Reply:
part of good testing is recognizing equivalence cases and uses those to prune the exploration space. I don't see why at rest is not an equivalence case with reload. what is different?

Reply:
> part of good testing is recognizing equivalence cases and using those to prune the exploration space.

That's a good articulation of this concept. I will keep it in mind :-).

Let me correct myself - The difference is not the document at rest part. Rather, its how long has the document been at rest. Whether the document has been at rest longer than the "sweep" threshold or not is an important factor w.r.t. GC.
If this time is longer than the sweep threshold, GC would immediately delete objects that are unreferenced. However, trailing ops may change this and GC has to take it into account. This would not happen when the rest time is smaller than the sweep threshold.

Having said that, I do see you point and I think the existing test may capture this scenario - When a container reloads, the summary may contain objects that were unreferenced a while back and are now eligible for sweep. This seems like the scenario I am explicitly trying to test but I don't need to.

Reply:
not blocking this, but not going to approve, since its a test branch :) consider adding a single case for at rest, so we can detect if the equivalence class no longer holds, and we can keep stress as simple as possible, while still maintaining good coverage.

Reply:
Thanks Tony, appreciate the feedback :-).

>consider adding a single case for at rest, so we can detect if the equivalence class no longer holds

Not sure what you mean by this. Can you please explain this a little?

Reply:
if you think the case of truly at rest could ever possibly be different than reload, then adding a single test case to cover it would satisfy that we have coverage for that case and could detect regression, but it wouldn't be necessary to add to stress as they are equivalence cases today. It probably does matter as i can't see a situation where at rest and reload are different unless the ordering service becomes GC aware, and does something at session close aka at no more clients aka at rest. that would be such a big change we probably don't need to worry about it as it would require a whole bunch of testing if it ever made sense to do, which is probably won't.